src/main/java/com/google/api/generator/engine/ast/MethodDefinition.java

```diff
@@ -34,7 +34,9 @@
   // Required.
   public abstract IdentifierNode methodIdentifier();
 
-  // TODO(xiahzhenliu): Add a Nullable JavaDoc here.
+  @Nullable
+  public abstract ImmutableList<CommentStatement> commentStatements();
```

Comment:
Let's call this `headerCommentStatements` or similar.

Reply:
Done. Thanks